SPECS/audit/audit.spec

```diff
@@ -170,9 +166,9 @@ make %{?_smp_mflags} check
 %{python3_sitelib}/*
 
 %changelog
-*   Fri Feb 18 2022 Thomas Crain <thcrain@microsoft.com> - 3.0-12
--   Bump release to force rebuild with golang 1.16.14
-
+*   Tue Feb 01 2022 Max Brodeur-Urbas <maxbr@microsoft.com> - 3.0-12
```

Comment:
This looks to be overwriting a previous changelog entry- please fix and bump the spec to release 13

Reply:
I tried to include the overwritten changelog as a referenced change in my own log but you're right this makes more sense. Fixing now. Thanks!